### ISSUE TITLE
Track events that got previously tracked on Dante

### ISSUE THREAD

**Issue by shockbytes** [shockbytes/DanteX#65]
### Overview

Simply put, track all events that were previously tracked on Dante, which are located in this file: https://github.com/shockbytes/Dante/blob/develop/tracking/src/main/java/at/shockbytes/tracking/event/DanteTrackingEvent.kt

### Implementation

The tracker should have a debug and a production implementation, just as done in Dante. For debug purposes, all should be tracked to the console. In production, Firebase Analytics should be used. https://github.com/shockbytes/Dante/tree/develop/tracking/src/main/java/at/shockbytes/tracking

Simply put, just port the Kotlin implementation to Dart.
